docs/features/connections.md

```diff
@@ -10,19 +10,20 @@ keywords:
   - "Online connections"
   - "Networking opportunities"
   - "tech connections"
-  - "grow newtwork"
+  - "grow network"
   - "grow connection"
   - "Community connections"
-
 ---
 
 ## What are Connections?
 
-Connections on OpenSauced serve as a way to build and nurture your professional network within the open source ecosystem. Imagine you're searching for experts in Artificial Intelligence using our [Contributor Insights](contributor-insights.md) feature, and you come across several promising developers. What's next? With Connections, you can now send a request to connect, bringing these experts into your professional circle for easy communication and collaboration.
+Connections on OpenSauced serve as a way to build and nurture your professional network within the open source ecosystem. Imagine you're searching for experts in Artificial Intelligence using our [Contributor Insights](contributor-insights.md) feature, and you come across several promising developers. What's next?
```

Comment:
[improvement] I think the wording here could be improved. It’s already been established that the features are apart of OpenSauced, so I think it is best to be a bit more straightforward at the beginning of this paragraph. Thoughts? 
```suggestion
This feature can help you build and nurture your professional network within the open source ecosystem. Imagine you're searching for experts in Artificial Intelligence using our [Contributor Insights](contributor-insights.md) feature, and you come across several promising developers. What's next?
```

Reply:
We don't make any content changes here as we want to leave out this section for now.
The only change here was a minor format.

But thanks for the input. 👍